BP/_BP_design.info

```diff
@@ -0,0 +1,20 @@
+Notes:
+Remember you changed some dharma localization (custom localization) stuff
+Fixup commented add_pressed_claims in history/characters
+Fix all localizations of type:
+	FIX_ME
+	blank_return
+	All in localization files of type _FIXME_*_l_english.yml
+
+Files that MUST replace Vanilla files:
+
+common/casus_belli_types/00_event_war.txt - Removal of hungarian migration reference
+common/cultures/innovations/00_<era>_innovations.txt (We changed some fundamental innovations here)
+common/defines/00_defines.txt (Some overwrites, don't know if we can do a soft overwrite like other keys)
+common/defines/graphic/00_graphics.txt (Some overwrites, don't know if we can do a soft overwrite like other keys)
+common/scripted_character_templates/00_pool_repopulation_character_templates.txt (1 error remains if we don't)
+
+events/decisions_events/east_europe_events.txt : Removal of bosnian_church, hungarian migration references
+events/game_rule_events.txt : Removal of hungarian migration references
+
+localization/english/load_tips_l_english.yml (Overwriting vanilla load tips)
```

Comment:
It is unclear if this file is you listing notes for yourself or for others. If so iut is better to include that sort of comment directly in the files.

Reply:
It's both; I include comments within the file for what changes between Vanilla and BP, but I've found it helpful to have a single file listing everything as well.

Reply:
Ok, no problems